docs/docs/native-docker.md

```diff
@@ -1,23 +1,70 @@
 ---
-title: Running Containers on Marathon
+title: Provisioning Containers
 ---
 
-Marathon supports Docker and Appc container images by either one of two runtimes; Docker Engine and Universal Container Runtime.
+# Provisioning Containers
 
-# Running Docker Containers on Marathon
+A containerizer is a Mesos agent component responsible for launching containers, within which you can run a Marathon app. Running apps in containers offers a number of benefits, including the ability to isolate tasks from one another and control task resources programmatically.
 
-Marathon enables users to run Docker container images with two different runtimes:
+Marathon enables users to run container images with two different runtimes:
 
-  1. [Docker containerizer](# Docker Containerizer) using the native Docker Engine as runtime.
-  2. Mesos containerizer using the Universal Container Runtime.  
+1. Mesos containerizer using the [Universal Container Runtime](#ucr).
+1. [Docker containerizer](#docker-containerizer) using the native Docker Engine as runtime.
```

Comment:
Thanks for looking this over, @janisz -- markdown will render lists in numerical order if you start each item with 1. This way, if the list is modified, you don't have to renumber.